lxd/db/node.go

```diff
@@ -504,6 +504,23 @@ func (c *ClusterTx) NodeUpdateVersion(id int64, version [2]int) error {
 	return nil
 }
 
+// NodeGetImages returns all images that the current LXD node instance has.
+func (c *ClusterTx) NodeGetImages() ([]string, error) {
+	return c.NodeGetImagesByID(c.nodeID)
+}
+
+// NodeGetImagesByID returns all images that the LXD node instance has with the given node id.
+func (c *ClusterTx) NodeGetImagesByID(id int64) ([]string, error) {
```

Comment:
Please move this API to lxd/db/images.go, since node.go should not look at the images table.